### PR TITLE
added fallback to an amorphic set secretSession and added body-parser…

### PR DESCRIPTION
**Pull request by nikmash**
… limit as a config value

@CSMastermind @hackerrdave please review!

### CODE REVIEW

File: lib/buildStartUpParams.js [hunk 1]
```diff
@@ -19,11 +19,17 @@ function buildStartUpParams(configStore) {
...
     amorphicOptions.appList = rootCfg.get('applications');
     amorphicOptions.appStartList = rootCfg.get('application').split(';');
     amorphicOptions.mainApp = amorphicOptions.appStartList[0];
     amorphicOptions.port = rootCfg.get('port');
+
+    if (rootCfg.get('sessionSecret')) {
+        amorphicOptions.sessionSecret = rootCfg.get('sessionSecret');
+    } else {
+        amorphicOptions.sessionSecret = 'swat_team';
+        console.warn('WARNING you are starting amorphic with no sessionSecret. The default sessionSecret "swat_team" will be used.');
```
Comment (hackerrdave) [inline lib/buildStartUpParams.js]: should we use amorphic logger for this?

File: lib/startUpServer.js [hunk 2]
```diff
@@ -39,6 +39,8 @@ function startUpServer(preSessionInject, postSessionInject, appList, appStartLis
     let amorphicOptions = AmorphicContext.amorphicOptions;
     let mainApp = amorphicOptions.mainApp;
     let appContext = AmorphicContext.appContext;
+    let appConfig = AmorphicContext.applicationConfig[mainApp] || {};
```
Comment (hackerrdave) [inline lib/startUpServer.js]: do we ever want to fall back to empty object here?

File: lib/buildStartUpParams.js (hunk 1 above)
Comment (nikmash) [reply lib/buildStartUpParams.js]: I don't think amorphic logger is instantiated at this point. Which is why I used a `console.warn`

File: lib/startUpServer.js (hunk 2 above)
Comment (nikmash) [reply lib/startUpServer.js]: good point I don't think so I was following the pattern elsewhere in the codebase.

**Comment by CSMastermind** [comment]
Need to update `README.md`.  Otherwise, this looks good.

File: lib/buildStartUpParams.js (hunk 1 above)
Comment (CSMastermind) [reply lib/buildStartUpParams.js]: There's an internal logging class you can use.  Check out `/lib/utils/logger.js`.

**Pull request merged**